continuous-pipe.yml

```diff
@@ -142,5 +139,8 @@ tasks:
         symfony_php56_apache:
           image: quay.io/continuouspipe/symfony-php5.6-apache
           tag: latest
+        varnish:
+          image: quay.io/continuouspipe/varnish4
```

Comment:
`quay.io/continuouspipe/drupal8-varnish4` and `quay.io/continuouspipe/magento2-varnish4` require this base image.